hid-xpadneo/src/etc-udev-rules.d/98-xpadneo.rules

```diff
@@ -0,0 +1,2 @@
+ACTION=="add", KERNEL=="0005:045E:02FD.*|0005:045E:02E0.*|0005:045E:0B05.*", SUBSYSTEM=="hid", DRIVER!="xpadneo", ATTR{driver/unbind}="%k", ATTR{[drivers/hid:xpadneo]bind}="%k"
+ACTION=="add", KERNEL=="js*", DRIVERS=="xpadneo", RUN+="/usr/bin/jscal -u 8,0,1,3,4,2,5,16,17,10,304,305,307,308,310,311,314,315,317,318"
```

Comment:
This is not compatible with the Chrome Gamepad API as it tries to apply the mapping from `evdev` to `joydev`. Maybe better move to documentation instead.

Reply:
Moved to patch queue for consideration.